code/_onclick/human.dm

```diff
@@ -10,8 +10,8 @@
 	var/last_chew = 0
 
 /mob/living/carbon/human/click(var/atom/A, var/list/mods)
-	if(interactee)
-		return interactee.handle_click(src, A, mods)
+	if(interactee?.handle_click(src, A, mods))
+		return
```

Comment:
so no longer returning a value?

Reply:
Well, better play safe and make it return a value then.